[31m[1mERROR   [0m sgqlc.endpoint.http:base.py:213 GraphQL query failed with 1 errors
[32mINFO    [0m sgqlc.endpoint.http:base.py:220 Error #0:
[32mINFO    [0m sgqlc.endpoint.http:base.py:222    | No row was found when one was required